pkg/apis/autoscaling/v1alpha1/pa_types.go

```diff
@@ -80,6 +80,9 @@ type PodAutoscalerSpec struct {
 	// is responsible for quickly right-sizing.
 	ScaleTargetRef corev1.ObjectReference `json:"scaleTargetRef"`
 
+	// Reachable specifies whether or not the `ScaleTargetRef` can be reached.
+	Reachable bool `json:"reachable"`
```

Comment:
One concern I have is that on the upgrade path from 0.8 reachable things will appear as `false` until they've been reconciled and potentially dip below the minimum availability.  What if we make this `*bool` (and `omitempty`), treat unspecified as unknown and bias towards the annotation when unknown?

Perhaps we can also use this initial unknown state to trigger the `minScale` provisioning at initial deployment (my comment above), and only after the Revision's become Ready reflect reachability here...  (thinking out loud)

Reply:
Sounds good. I'll break this out into a separate PR

Reply:
Sounds good, LMK when you have it!

Reply:
@mattmoor #5147